### ISSUE TITLE
Not working with Ubuntu 18.04

### ISSUE THREAD

**Issue by ecerroni** [tvanro/prerender-alpine#3]
This does not seem to work with Ubuntu 18.04. Running the container always fails:
```
root@prerender-test:~# docker run --name server.prerender tvanro/prerender-alpine:5.5.1
2019-04-02T07:38:02.391Z Starting Prerender
2019-04-02T07:38:02.394Z Starting Chrome
2019-04-02T07:38:02.405Z Prerender server accepting requests on port 3000
2019-04-02T07:38:02.422Z Chrome connection closed... restarting Chrome
2019-04-02T07:38:02.423Z Chrome died immediately after restart... stopping Prerender
```

It works with Ubuntu 16.04 though.

It also seems to share the issue with other similar docker images using Headless Chrome

**Comment by tvanro** [comment]
Hi @ecerroni,
I tested this as well on my side in a VM with Ubuntu 18.04 but I cannot reproduce this issue.

```
tom@tom-VirtualBox:~$ docker -v
Docker version 18.09.0, build 4d60db4
tom@tom-VirtualBox:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
tom@tom-VirtualBox:~$ docker run --name server.prerender tvanro/prerender-alpine:5.5.1
2019-04-03T01:20:54.126Z Starting Prerender
2019-04-03T01:20:54.128Z Starting Chrome
2019-04-03T01:20:54.140Z Prerender server accepting requests on port 3000
2019-04-03T01:20:54.655Z Started Chrome: HeadlessChrome/68.0.3440.75
```

There are however some related issues in the main prerender repository. Someone resolved their issue by running it as a non-root user: https://github.com/prerender/prerender/issues/450#issuecomment-414683227

Also, can you have a try with the latest version?  
The Chromium version has been updated to 72 in `tvanro/prerender-alpine:5.6.0`.

**Comment by ecerroni** [comment]
Yes, I can confirm that version `5.6.0` solved the issue on my VM (both 18.04 and 18.10)

Thanks!
